test/Driver/minimal_make_dependencies.swift

```diff
@@ -0,0 +1,63 @@
+// Try WMO - we need to emit targets for each primary object file.
```

Comment:
Would be nice to add a comment at the top of the file summarizing what is being tested.